src/main/resources/tags/climbing_pick_weak_surfaces.json

```diff
@@ -2,6 +2,8 @@
 	"values" : [
 		"#slimefun:concrete_powders",
 		"#minecraft:sand",
-		"minecraft:gravel"
+		"minecraft:gravel",
+        "minecraft:skulk",
+        "minecraft:clay"
```

Comment:
This will break all versions below 1.18.
see how blackstone is added.
Change them all to that.

Also formatting you are using spaces instead of tabs.